### DISCUSSION TITLE
E1 Zoom - Link to v716_21112408

### DISCUSSION THREAD

**Discussion by lscorcia** [AT0myks/reolink-fw-archive#10] [Firmware sharing]
Hi, another old firmware for E1 Zoom - unfortunately I only have a copy and not an official link but I think I got it from the official webserver initially.
https://drive.google.com/file/d/1zRlbf0NnmlBt3gFXcTFVTGvrEp9q0qT5/view?usp=sharing

Please rehost it as I will have to remove the link in a week or so.
Thanks for your work!

**Comment by AT0myks** [comment]
Thanks for providing 2 missing firmwares, I added this one to my drive.
I've moved your issues to the discussions, where the firmware sharing takes place.
